README.md

```diff
@@ -2,10 +2,9 @@
 
 In this project i acheived the following
 
-- Used ES6 modules, with import and export
-- Created an npm project with webpack
-- Wrote plain HTML markup with minimum styling
-- Used Gitflow in this project
+- improved the look and feel of the application, adding the styles of your choice.
+- Maintained the general layout of the wireframe
+- Used plain CSS to style the page
 
 ![screenshot](./images/screenshot.JPG)
 
```

Comment:
- [ ] In your `##Usage` to clone project section, instead of saying this open the `index.html`, I will recommend writing something like `npm i` and '`npm run start`'..., this way you give clearly instructions of all the steps, think that your project could be seen for someone with no coding knowledge :smile:. Also, include the code on how to clone, instead of just writing.

Reply:
Thanks for the corrections